go/service/main.go

```diff
@@ -227,6 +233,18 @@ func (d *Service) checkTrackingEveryHour() {
 	}()
 }
 
+func (d *Service) gregordConnect() error {
+	h := newGregorHandler(d.G())
+	d.G().Log.Debug("gregor URI: %s", d.G().Env.GetGregorURI())
+	rootCert, err := ioutil.ReadFile(os.Getenv("GREGOR_ROOT_CERT"))
```

Comment:
This should go in `ca.go` and should follow the same template as the one we have for the api server. In fact, it will almost certainly be the same root CA.